### PR TITLE
Feature/name in sketch toolkit widget

### PR DESCRIPTION
**Pull request by m7kvqbe1**
## Related Jira issue:

https://rn-nelson.atlassian.net/browse/EN-597

## Overview

Implemented the docs-site SketchWidget component.

## Reason

>For each component (in design documentation) we have a corresponding symbol in the Sketch library. This element informs the user in what the symbol is named.

## Work carried out

- [x] Implement component within the docs-site codebase
- [x] Add automated tests
- [x] Perform UAT using BrowserStack down to Chrome 38 (and a range of form factors)

## Screenshot

Screenshot 2019-05-31 at 14 05 30
Screenshot 2019-05-31 at 14 50 49

### CODE REVIEW

File: packages/docs-site/src/components/presenters/sketch-widget/download-icon.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg version="1.1" viewBox="0 0 12 13" xmlns="http://www.w3.org/2000/svg">
+<title>Download Icon</title>
+<desc>Created with Sketch.</desc>
```
Comment (antictor) [inline packages/docs-site/src/components/presenters/sketch-widget/download-icon.svg]: I'd remove the created with sketch but and the id's that never get used

File: packages/docs-site/src/components/presenters/sketch-widget/index.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import PropTypes from 'prop-types'
+
+import './sketch-widget.scss'
+
+import SketchLogo from './sketch-logo.svg'
+import DownloadIcon from './download-icon.svg'
+
+const SketchWidget = ({ name, href }) => {
+  return (
+    <article className="sketch-widget">
```
Comment (antictor) [inline packages/docs-site/src/components/presenters/sketch-widget/index.js:11]: I left the rn- part of the classname in footer. wasn't sure if we wanted to namespace or not. doesn't matter

File: packages/docs-site/src/components/presenters/sketch-widget/index.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+          href={href}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <DownloadIcon />
+          Download Sketch toolkit
+        </a>
+      </div>
+      <div className="sketch-widget__body">
+        <SketchLogo className="sketch-widget__sketch-logo" />
```
Comment (antictor) [inline packages/docs-site/src/components/presenters/sketch-widget/index.js:26]: ooo I didn't know I could attach css classes to svg's this way

File: packages/docs-site/src/components/presenters/sketch-widget/index.test.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+
+import SketchWidget from './index'
+
+describe('SketchWidget', () => {
+  let sketchwidget
+
+  describe('when the SketchWidget is generated with name and href props', () => {
+    beforeEach(() => {
```
Comment (antictor) [inline packages/docs-site/src/components/presenters/sketch-widget/index.test.js:10]: 👍

File: packages/docs-site/src/components/presenters/sketch-widget/sketch-widget.scss [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+  line-height: $head-line-height;
+  margin-bottom: spacing(4);
+}
+
+.sketch-widget__link {
+  line-height: $head-line-height;
+  color: #3a8fdd;
+  text-decoration: none;
+
+  svg {
```
Comment (antictor) [inline packages/docs-site/src/components/presenters/sketch-widget/sketch-widget.scss:33]: do we want to style by tag name? would it be better to use a class name?

File: packages/docs-site/src/components/presenters/sketch-widget/sketch-widget.scss [hunk 6]
```diff
@@ -0,0 +1,77 @@
+@import "@royalnavy/css-framework";
+
+$border-radius: 4px;
```
Comment (antictor) [inline packages/docs-site/src/components/presenters/sketch-widget/sketch-widget.scss:3]: wonder if we should have a common border radius or if that's over the top

**Review by antictor — APPROVED** [review]
LGTM, just a few minor comments

File: packages/docs-site/src/components/presenters/sketch-widget/index.js (hunk 2 above)
Comment (m7kvqbe1) [reply packages/docs-site/src/components/presenters/sketch-widget/index.js:11]: I wanted to keep a distinction between the namespaced component-library styles and those within this codebase.

File: packages/docs-site/src/components/presenters/sketch-widget/download-icon.svg (hunk 1 above)
Comment (m7kvqbe1) [reply packages/docs-site/src/components/presenters/sketch-widget/download-icon.svg]: Yeah fair enough - put it through an SVG minifier but for some reason it didn't strip this.

File: packages/docs-site/src/components/presenters/sketch-widget/sketch-widget.scss (hunk 5 above)
Comment (m7kvqbe1) [reply packages/docs-site/src/components/presenters/sketch-widget/sketch-widget.scss:33]: Hmm well I coupled it very closely to the parent element. This is up for debate I think.

File: packages/docs-site/src/components/presenters/sketch-widget/sketch-widget.scss (hunk 6 above)
Comment (m7kvqbe1) [reply packages/docs-site/src/components/presenters/sketch-widget/sketch-widget.scss:3]: As we implement a few more components I think common would be good - i'm sure there are probably edge cases though - Lets pick designer brains.

**Pull request merged**